src/Text/Text.stories.tsx

```diff
@@ -30,3 +90,5 @@ WithTitle.args = {
   tag: 'p',
   title: 'a11y title tooltip',
 }
+
+export const TypographyCollection = TypographyTemplate.bind({})
```

Comment:
No need for Typography part? We're already in the Text part, we can call it "Collection"